ui/components/ui/tooltip/index.scss

```diff
@@ -2,7 +2,7 @@
 
 .tippy-popper {
   .tippy-tooltip.tippy-tooltip--mm-custom-theme {
-    background: var(--color-background-muted);
+    background: var(--color-background-section);
```

Comment:
Updating all instances of `--color-background-muted` to `--color-background-section`. The remaining uses are correctly applied to interactive elements(notice the hover and pressed states).

<img width="475" alt="Screenshot 2025-06-19 at 5 38 09 PM" src="https://github.com/user-attachments/assets/238be8c1-b5eb-477d-9c93-6daed41f7607" />